webapp/messages/es.json

```diff
@@ -33,6 +35,7 @@
       "pending": "Pendiente",
       "rejected": "Tx rechazada"
     },
+    "try-again": "Try again",
```

Comment:
```suggestion
    "try-again": "Intente nuevamente",
```

Reply:
😅  Fixed in [c267a8f](https://github.com/hemilabs/ui-monorepo/pull/849/commits/c267a8ffa3d18c00e327d2eee9643e499d1d4615)